src/curl_async.c

```diff
@@ -258,7 +258,7 @@ static int perform_curl_io_and_complete(struct swupd_curl_parallel_handle *h, in
 		 * completing the download encounters further problems. */
 		curl_ret = swupd_download_file_close(msg->data.result, &file->file);
 		file->status = process_curl_error_codes(curl_ret, handle, &err);
-		debug("CURL - complete ASYNC download: %s -> %s, status=%d\n", file->url, file->file.path, file->status);
+		debug("Curl: complete ASYNC download: %s -> %s, status=%d\n", file->url, file->file.path, file->status);
```

Comment:
The messages in this section now use "Curl: ..." consistently, should we stick to this format here or should we go with the "curl - ..." format?

Reply:
Me too. I'll fix that.

Reply:
oops, I commented in the wrong question